validphys2/src/validphys/promptutils.py

```diff
@@ -57,3 +57,23 @@ def nothing(event):
     complete_message = merge_formatted_text([message, yes_no_str(default)])
     session = PromptSession(complete_message, key_bindings=bindings)
     return session.prompt()
+
+#We need some sort of cache because prompt_toolkit calls the callable
+#every time it tries to complete.
+class KeywordsWithCache():
+    def __init__(self, loader):
+        self.loader = loader
+        self.words = None
+
+    def __call__(self):
+        if self.words is None:
+            if hasattr(self.loader, "remote_keywords"):
```

Comment:
Why is this? At the moment if someone uses this with anything other than a `RemoteLoader` they're probably doing something strange/wrong, so maybe it should just error in such a case.

Reply:
`vp-comparefits` could be using a `Loader` rather than a `FallbackLoader` if you pass the `--no-net` flag. But fine, I'll make that more explicit.